### PR TITLE
[MAINTENANCE] Adjust reading of page uid in controller

### PR DESCRIPTION
**Pull request by beatrycze-volk**

### CODE REVIEW

File: Classes/Controller/AbstractController.php [hunk 1]
```diff
@@ -91,6 +97,7 @@ public function injectDocumentRepository(DocumentRepository $documentRepository)
     protected function initialize(): void
     {
         $this->requestData = GeneralUtility::_GPmerged('tx_dlf');
+        $this->pageUid = (int) GeneralUtility::_GET('id');
```
Comment (sebastian-meyer) [inline Classes/Controller/AbstractController.php:100]: Does this also work when using URL enhancers (rewriting the `id` parameter to a "speaking" path segment)?

Comment (beatrycze-volk) [reply Classes/Controller/AbstractController.php:100]: I can't test it currently, as my DDEV instance is in the process of migration to PHP 8.2 and TYPO3 11. I was not yet able to configure routing correctly and need to use URLS as `https://sachsendigital.ddev.site/index.php?id=5`. I will mark it as draft and check it out after routing is configured.

Comment (sebastian-meyer) [reply Classes/Controller/AbstractController.php:100]: Thank you!

Comment (beatrycze-volk) [reply Classes/Controller/AbstractController.php:100]: I have tested it now. It works also with URL enhancers :)

Comment (sebastian-meyer) [reply Classes/Controller/AbstractController.php:100]: Great!

**Review by sebastian-meyer — APPROVED** [review]

**Pull request merged**